File: Gabaritos/N1_2_N2/C++/media_mediana.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int main(){
    int a, b;
    vector<int> c;
    int m =0;
    cin >> a;
    cin >> b;

    //SENDO A MEDIA E MEDIANA = A
    m = (2*a) - b;
    c.push_back(m);

    //SENDO A MEDIA E MEDIANA = B
    m = (2*b) - a;
    c.push_back(m);

    //SENDO A MEDIA E MEDIANA IGUAL C
    m = int((a+b)/2);
    c.push_back(m);

    sort(c.begin(), c.end());
    cout<<c[0]<<endl;
}

File: Gabaritos/media_mediana.py
a, b = map(int, input().split())
c = []

#SENDO A MEDIA E MEDIANA IGUAL A
m = (2*a) - b
c.append(m)

#SENDO A MEDIA E MEDIANA IGUAL B
m = (2*b) - a
c.append(m)

#SENDO A MEDIA E MEDIANA IGUAL C
m = (a+b)//2
c.append(m)

c.sort()
print(c[0])